src/app/layout.tsx

```diff
@@ -20,7 +21,9 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className={inter.className}>
-        <SessionContext>{children}</SessionContext>
+        <SessionContext>
+          <SiteLayout>{children}</SiteLayout>
```

Comment:
The site layout is currently applied to all pages. We should only use it in the landing page and FAQ page, not as a global layout.

Reply:
fixed - please check.